test/fixtures/scrub.txt

```diff
@@ -1 +1,2 @@
-http://whatismyipaddress.com/
+http://geoip.ubuntu.com/lookup
+https://ip-info.ff.avast.com/v1/info
```

Comment:
Replace the originally-used ip-leaking service with other two services. Motivation for the replacement: the one that we're removing has apparently banned github and we need to use other services. One of the two services is also the one we use for detecting the user IP (i.e. geoip.ubuntu.com). However, it also returns a web page rather than a JSON. So, I want basically to test for scrubbing in more cases, and I'm fine with using geoip.ubuntu.com also for testing scrubbing because we are using more than one service for testing scrubbing anyway.